assets/js/components/AssetViewer.js

```diff
@@ -84,19 +84,19 @@ module.exports = class AssetViewer {
     if (!this.loader) {
       this.loader = Promise.all([
         utils.loadJavaScript(
-          'https://cdnjs.cloudflare.com/ajax/libs/photoswipe/4.1.3/photoswipe.min.js',
+          'https://cdnjs.cloudflare.com/ajax/libs/photoswipe/4.1.2/photoswipe.min.js',
```

Comment:
Any reason why this library is being downgraded?

Reply:
These changes have come in from  updating from last September to current. Not sure why this has been downgraded but will update again